homeassistant/components/prometheus/__init__.py

```diff
@@ -650,8 +652,9 @@ class PrometheusView(HomeAssistantView):
     url = API_ENDPOINT
     name = "api:prometheus"
 
-    def __init__(self, prometheus_cli):
-        """Initialize Prometheus view."""
+    def __init__(self, prometheus_cli, config:ConfigType):
+        """Initialize Prometheus view."""        
+        self.requires_auth =  config[CONF_API_AUTHENTICATION]
```

Comment:
Double whitespace.
It seems that you skipped the pre-commit hooks :(

Reply:
done